### PR TITLE
HTML-JAVASCRIPT_CSS by Ivan Genchev -Task One

### PR DESCRIPTION
**Pull request by freekman**
1)Task one

### CODE REVIEW

File: src/task1/SampleMaven.txt [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (mgenov) [inline src/task1/SampleMaven.txt]: Not sure that this file belongs here.

Comment (freekman) [inline src/task1/SampleMaven.txt]: It is just a test (txt ) file for the purpose of the task.I can rename it to testFile.txt if it is bether.

**Comment by mgenov** [comment]
I've added development tools information in the Required Reading section in the documentation: 
https://github.com/clouway/teamspace/wiki/HTML-&-JavaScript-&-CSS

**Comment by mgenov** [comment]
LGTM !

**Pull request merged**